chart/compass/values.yaml

```diff
@@ -588,6 +589,7 @@ global:
       pageSize: 200
       sizeParam: "$top"
       skipParam: "$skip"
+    containerName: "system-fetcher"
```

Comment:
Make use of this in `system-fetcher-job.yaml` where the container name is defined, otherwise this is double maintenance